src/ua.skrypnyk.hw10/Circle.java

```diff
@@ -0,0 +1,48 @@
+package src.ua.skrypnyk.hw10;
+
+import java.util.Scanner;
+
+public class Circle extends Shape {
+    private double perimeter;
+    private double area;
+    private int radius;
+
+    public Circle(double perimeter, double area) {
+        this.area = area;
+        this.perimeter = perimeter;
+    }
+    @Override
+    public double getPerimeter() {
+        return perimeter;
+    }
+    @Override
+    public double getArea() {
+        return area;
+    }
+    @Override
+    public double area() {
```

Comment:
Please move input logic out of method
Area method should only calculate an area